src/App.jsx

```diff
@@ -1,33 +1,56 @@
 import './App.scss';
+import { useState } from 'react';
+
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getPreparedMovies(movies, query) {
+  const preparedMovies = [...moviesFromServer];
+
+  if (query) {
+    return preparedMovies.filter(el => (
+      el.title.toLowerCase().includes(query.trim().toLowerCase())
```

Comment:
```suggestion
      el.title.toLowerCase().includes(normalizedQuery)
```
you are doing the same twice - save it into a variable and reuse

Reply:
done